lib/primer/view_components/version.rb

```diff
@@ -11,3 +11,7 @@ module VERSION
     end
   end
 end
+
+if __FILE__ == $0
+  puts Primer::ViewComponents::VERSION::STRING
+end
```

Comment:
```suggestion
puts Primer::ViewComponents::VERSION::STRING if __FILE__ == $PROGRAM_NAME
```

let's make rubocop happy 😄